src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -47,13 +48,43 @@ class IOUAmountPage extends React.Component {
 
         this.updateAmountIfValidInput = this.updateAmountIfValidInput.bind(this);
         this.state = {
-            amount: '',
+            amount: props.selectedAmount || '',
         };
     }
 
     componentDidMount() {
         if (this.textInput) {
-            this.textInput.focus();
+            // Workaround: Input doesn't focus without a setTimeout
+            setTimeout(() => this.textInput.focus(), 300);
```

Comment:
Is there no other way to do this without a setTimeout?

Reply:
Updated! Looks like the iou not being done loading was the issue - the componentDidMount ran before the iou was actually finished loading so I had to use a componentDidMount to watch for the prop change and then focus on the input